Performance/QPSBenchmark/Sources/QPSBenchmark/Runtime/AsyncClient.swift

```diff
@@ -207,12 +207,14 @@ final class AsyncQPSClient<RequestMakerType: RequestMaker>: QPSClient {
 
     /// Returns if it is permissible to make another request - ie we've not been asked to stop, and we're not at the limit of outstanding requests.
     private var canMakeRequest: Bool {
+      assert(self.connection.eventLoop.inEventLoop)
```

Comment:
As per Cory's comment offline: let's use `assertInEventLoop()` and `preconditionInEventLoop()`